spec/game_spec.rb

```diff
@@ -15,21 +15,45 @@
     @game_3 = games[11] 
     @game_4 = games[5] 
     @game_5 = games[12]
+    @game_6 = games[9]
   end
 
   it "exists" do
     expect(@game_1).to be_a(Game)
   end
 
-  it "has game_id, season, season_type, away_team_id, home_team_id, away_goals, home_goals" do
```

Comment:
I'm wondering if we should put this into a describe block for readability?

Thank you for separating out the tests!

Reply:
Referring to the attribute tests

Reply:
I like that idea! If we do describe blocks in this spec, we should do them in all specs for consistency. I suggest adding this to our Miro board for our "when we have time" category so we can check that all our specs have describe blocks